server/services/overviewPageService.test.ts

```diff
@@ -773,43 +772,6 @@ describe('OverviewPageService', () => {
         activeAlertCount: 1,
         nonAssociationsCount: 2,
         nonAssociationsUrl: `${config.serviceUrls.nonAssociations}/prisoner/G6123VU/non-associations`,
-        showNonAssociationsLink: false,
-      })
-    })
-
-    describe('Link to non-associations', () => {
```

Comment:
This test was checking whether we were expecting the non-association link to be shown or not based on the setting with the list of private beta prisons.

But because now the link is always shown this test doesn't make sense anymore (and the integration test now checks for this link presence)